android/sample/src/main/kotlin/com/gu/source/IconsPreview.kt

```diff
@@ -0,0 +1,115 @@
+package com.gu.source
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.GridItemSpan
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.gu.source.components.buttons.SourceButton
+import com.gu.source.components.buttons.SourceIconButton
+import com.gu.source.daynight.AppColour
+import com.gu.source.daynight.AppColourMode
+import com.gu.source.icons.Check
+import com.gu.source.icons.ChevronLeft
+import com.gu.source.icons.ChevronRight
+import com.gu.source.icons.Minus
+import com.gu.source.icons.Plus
+import com.gu.source.presets.palette.Brand400
+import com.gu.source.presets.palette.Neutral0
+import com.gu.source.presets.palette.Neutral100
+import com.gu.source.presets.palette.Sport500
+import com.gu.source.presets.typography.HeadlineBold20
+import com.gu.source.presets.typography.TextSansBold14
+import com.gu.source.presets.typography.TextSansBold15
+
+private val icons = mapOf(
+    "Base" to listOf(
+        Source.Icons.Base.Check,
+        Source.Icons.Base.ChevronLeft,
+        Source.Icons.Base.ChevronRight,
+        Source.Icons.Base.Minus,
+        Source.Icons.Base.Plus,
+    ),
+)
+
+@Composable
+internal fun Icons(modifier: Modifier = Modifier) {
```

Comment:
Required: Rename this to `IconsPreview`. We're already using `Icons` as a namespace and repeating the name here could cause confusion.